apps/src/templates/courseOverview/resourceType.js

```diff
@@ -1,5 +1,3 @@
-import i18n from '@cdo/locale';
-
 // We want level builders to be able to specify which of these strings is used,
 // but then want to make sure to show teachers the localized version
 const ResourceType = {
```

Comment:
I think this might be able to be removed as well now, which means this whole file can just be deleted

Reply:
Thank you for the careful reviews!